packages/flutter_tools/lib/executable.dart

```diff
@@ -186,7 +195,12 @@ Future<Null> main(List<String> args) async {
 }
 
 Future<File> _createCrashReport(List<String> args, dynamic error, Chain chain) async {
-  FileSystem fs = const LocalFileSystem();
+  FileSystem fs = context?.getVariable(FileSystem);
```

Comment:
We were explicitly using `LocalFileSystem` (and not getting it out of `context`) to avoid trying to write your crash report to a `ReplayFileSystem`, which would then throw a secondary error because there's no such command in the replay logs.